azerothcore/azerothcore-wotlk
# fix(DB/SAI): Mirage Raceway - goblin / gnome racing

##### CHANGES PROPOSED:
- Implement racing between goblins and gnomes at the Mirage Raceway (Shimmering Flats). I did this from scratch as there exists only very sparse information about the race.
- Fix a few position issues of the goblin / gnome pit crew (otherwise they are randomly moving on tents / tables).

###### ISSUES ADDRESSED:
Closes #2012

##### TESTS PERFORMED:
tested successfully in-game

##### HOW TO TEST THE CHANGES:
- ```.tele MirageRaceway```
- the event will start ca. 30-90 seconds after Daisy is spawned (you'll see the Goblin / Gnome Racers slowly moving toward the start line)
- just follow the racers (goblin / gnome team will win randomly)
- after the race is finished the racers will despawn and after 10 minutes Daisy will also despawn
- a new race will start after Daisy respawns again (5 minutes respawn time)

##### KNOWN ISSUES AND TODO LIST:
none

##### Target branch(es):
Master
